contact-page.php

```diff
@@ -0,0 +1,219 @@
+			<textarea class="form__input form__input--no-right-space col-6" type="textarea" name="other_organisations" maxlength="3000" rows="8" laceholder="No more than 500 words please"></textarea>
+
+            <div class="form__row">
+                <div class="col-6">
+                    <label class="form__label" for="institution">Institution/Affiliation</label>
+        			<input class="form__input col-6" type="text" name="institution">
+                </div>
+
+                <div class="col-6">
+                    <label class="form__label" for="supervisors">Main supervisors (if appropriate)</label>
+                    <input class="form__input form__input--no-right-space col-6" type="text" name="supervisors">
+                </div>
+            </div>
+
+            <label class="form__label" for="outputs">Potential/predicted outputs (if appropriate)</label>
+			<textarea class="form__input col-6" type="textarea" name="outputs" maxlength="3000" rows="8" placeholder=""></textarea>
+
+            <label class="form__label" for="interest">Researchers interest in IMI</label>
+			<textarea class="form__input form__input--no-right-space col-6" type="textarea" name="interest" maxlength="3000" rows="8"></textarea>
+
+			<input class="form__btn btn btn--secondary btn--submit" type="submit" value="submit" name="submit">
+		</form>
+	</div>
+	<div class="js-form">
+        <form class="form grid js-contact-form">
+            <div class="form__row">
+                <div class="col-6">
+                    <label class="form__label" for="first-name">First Name</label>
+                    <input class="form__input col-6" type="text" id="first-name">
+                </div>
+                <div class="col-6">
+                    <label class="form__label" for="surname">Surname</label>
+                    <input class="form__input form__input--no-right-space col-6" type="text" id="surname">
+                </div>
+            </div>
+            <div class="form__row">
+                <div class="col-6">
+                    <label class="form__label" for="email">Email</label>
+                    <input class="form__input col-6" type="email" id="email">
+                </div>
+                <div class="col-6">
+                    <label class="form__label" for="contact-number">Contact Number</label>
+                    <input class="form__input form__input--no-right-space col-6" type="tel" id="number">
+                </div>
+            </div>
+            <label class="form__label" for="message">Your message</label>
+            <textarea class="form__input form__input--no-right-space col-12" maxlength="3000" rows="15" type="textarea" id="message"></textarea>
+            <input class="form__btn btn btn--secondary btn--submit" type="submit" name="submit">
+        </form>
+  	</div>
+  	<div class="js-form">
+        <form class="form grid js-research-form">
```

Comment:
The latter two media form and the research form are merely hardcoded at the moment. For now it is just the first form, Contact, under the contact tab. Just ignore these for now, but I would like to get them working in the same way if that's possible. @harryfinn